### ISSUE TITLE
Provide a JSON capability

### ISSUE THREAD

**Issue by millerf** [CirkusApp/airstrings#1]
It would be great to be able to `push`/`pull` JSON content directly. Maybe with a `--json` parameter...

**Comment by shpakovski** [comment]
Makes sense 🙇‍♂️ Let’s decide on the format, we do need to keep the order. Maybe something like this?
```json
[
  { "key": "…", "value": "…", "comment": null},
  { "key": "…", "value": "…", "comment": "…"},
  { "key": "…", "value": "…", "comment": "…"}
]
```

**Comment by millerf** [comment]
Unfortunately, [this](https://github.com/fwHub/cw-web/blob/d6026b9794ce13adc003480d6388fbfa0fa0075c/src/assets/locale/delete-user/en-US.lang.json) is our format so far.

```
{
  "KEY1": "...",
  "KEY2": "..."
}
```

I'll check if another format is supported natively by `https://github.com/ngx-translate`

**Comment by shpakovski** [comment]
Hi @millerf 👋 Do you approve my proposed JSON format?

_In addition, supporting YAML as well as JSON would be really cool. I updated the title._

**Comment by millerf** [comment]
yes, I can easily use those with a bit of magic :)
